lib/MumbleConnection.js

```diff
@@ -554,8 +611,15 @@ MumbleConnection.prototype._processAudio = function() {
             // We got listeners for voice event so do decode.
             var decoded = [];
             for( var f in packets ) {
+                var data;
+                if(this.opusEnabled) {
+                    data = packets[f].decoder.decode( packets[f].frame, this.FRAME_SIZE );
+                }
+                else {
+                    data = packets[f].decoder.decode( packets[f].frame );
+                }
                 decoded.push({
-                    data: packets[f].decoder.decode( packets[f].frame ),
+                    data: data,
                     codec: packets[f].codec,
```

Comment:
When the server decides to switch from Celt to Opus and vice-versa a ```CodecVersion``` message will be sent and all Codecs will be refreshed.